components/raftstore-v2/src/operation/command/admin/split.rs

```diff
@@ -365,6 +374,13 @@ impl<EK: KvEngine, ER: RaftEngine> Peer<EK, ER> {
             info!(self.logger, "not leader, skip.");
             return;
         }
+        // Check whether the admin request can be proposed when disk full.
```

Comment:
why it's repeated here.

Reply:
Maybe u mean the check in RequestSplit & RequestHalfSplit?
![image](https://github.com/tikv/tikv/assets/18441614/f8252043-ba31-4f70-ae9f-634ed7458653)
 
The checks on disk_full  are both needed.